docs/chains/darwinia-chains.md

```diff
@@ -13,10 +13,6 @@ Darwinia Chain (DC) is designed to be the primary chain for tokens and governanc
 DC is a public chain that can operate independently with its own consensus and security model, with its core business and application services, including cross-chain functionality of each application chain, controlled by 
 DC itself.
 
-#### Pangoro
```

Comment:
Why delete the pangoro here? This doc contains all darwinia chains including the testnets.

Reply:
![image](https://user-images.githubusercontent.com/11801722/180113669-37ba5f21-8ea8-4969-b095-ebdc73b61795.png)

Just follow the Crab network doc home page outline. Adding a separate section about the test chain here looks a little complicated. So I put the information about the Pangoro Smart Chain in the Darwinia Smart Chain introduction.